runelite-client/src/main/java/net/runelite/client/plugins/gpu/GpuPlugin.java

```diff
@@ -267,6 +270,10 @@ protected void startUp()
 				if (log.isDebugEnabled())
 				{
 					gl.glEnable(GL_DEBUG_OUTPUT);
+
+					// Need to control the debug messages here too.
+					gl.getContext().glDebugMessageControl(gl.GL_DEBUG_SOURCE_API, gl.GL_DEBUG_TYPE_OTHER,
```

Comment:
Are you sure it is necessary on both places? Because gl.getContext() == glContext, and so it should have the setting already set

Reply:
From testing it, it seems gl.getContext().glDebugMessageControl works but glContext.glDebugMessageControl doesn't so i removed it as it is unnecessary.